### DISCUSSION TITLE
Duplicates Removal | By Filename or Video Content?

### DISCUSSION THREAD

**Discussion by SheriffAbdullah** [TheLastGimbus/GooglePhotosTakeoutHelper#327] [General]
I want to know whether the duplicate files / photos are:
(i) removed based on filename, in this case, what are the odds that 2 different files don't have the same name?
(ii) removed based on the content or hashes of the file, in which case, it seems fine.

**Comment by TheLastGimbus** [comment]
gpth looks into file hash - two files must be *identical* to mark as dupliacte :ok_hand:
